packages/itmat-ui-react/src/components/users/createNewUser.tsx

```diff
@@ -44,7 +43,22 @@ export const CreateNewUser: React.FunctionComponent = () => {
         };
     }
 
-    if (completedCreationId) { return <Redirect to={`/users/${completedCreationId}`} />; }
+    if (completedCreation) {
+        return (
+            <div className={css.login_and_error_wrapper}>
+                <div className={`${css.login_box} appear_from_below`}>
+                    <h1>Registration Successful!</h1>
+                    <h2>Welcome {inputs.realName} to the IDEA-FAST project</h2>
+                    <br />
+                    <div>
+                        <p>Please check your email to setup the 2FA on Google Authenticator to log in.</p>
```

Comment:
Here again, lets not push Google Authenticator, we are not here to endorse.

Reply:
Agree, this OTP algorithm is standardised. Our mfa library works with other authenticator app as well. I just wanna give users a hint what app should be installed.